pants.ini

```diff
@@ -1,6 +1,6 @@
 [GLOBAL]
 pants_version: 1.15.0.dev3
-
 plugins: [
-    'pantsbuild.pants.contrib.go==%(pants_version)s',
-  ]
+	'pantsbuild.pants.contrib.go==%(pants_version)s',
```

Comment:
This changing is because of the use of `configparser`, right? This seems like an ok change then, since it seems like that module doesn't provide any options for spacing when writing out files.